go.mod

```diff
@@ -5,5 +5,5 @@ go 1.14
 require (
 	github.com/pkg/errors v0.9.1
 	github.com/stretchr/testify v1.4.0
-	golang.org/x/sys v0.0.0-20180810173357-98c5dad5d1a0
+	golang.org/x/sys v0.28.0
```

Comment:
> I can switch to v0.26.0 to keep it compatible.

I don't know enough about golang to agree/disagree.  I was just noticing a seemingly-unrelated deps change, because those types of changes have caused us issues in the C++ and Python world.

Reply:
I'll ask for some extra eyes on this PR from my team. Thanks for highlighting this!